### PR TITLE
WT-8421 Add a configure to allow importing files with timestamp smaller or equal to the stable timestamp

### PR DESCRIPTION
**Pull request by quchenhao**

### CODE REVIEW

File: dist/api_data.py [hunk 1]
```diff
@@ -1391,6 +1391,12 @@ def __ge__(self, other):
     Config('import', '', r'''
         configure import of an existing object into the currently running database''',
         type='category', subconfig=[
+        Config('compare_timestamp', 'oldest', r'''
```
Comment (kommiharibabu) [inline dist/api_data.py:1394]: I feel `compare_timestamp` name doesn't feel good, how about just `timestamp`?

File: src/schema/schema_create.c [hunk 2]
```diff
@@ -70,24 +77,25 @@ __check_imported_ts(WT_SESSION_IMPL *session, const char *uri, const char *confi
...
     WT_CKPT_FOREACH (ckptbase, ckpt) {
-        if (ckpt->ta.newest_start_durable_ts > txn_global->oldest_timestamp)
+        if (ckpt->ta.newest_start_durable_ts > ts)
             WT_ERR_MSG(session, EINVAL,
               "%s: import found aggregated newest start durable timestamp newer than the current "
-              "oldest timestamp, newest_start_durable_ts=%" PRIu64 ", oldest_ts=%" PRIu64,
-              uri, ckpt->ta.newest_start_durable_ts, txn_global->oldest_timestamp);
+              "%s timestamp, newest_start_durable_ts=%" PRIu64 ", %s_ts=%" PRIu64,
+              uri, ts_name, ckpt->ta.newest_start_durable_ts, ts_name,
+              txn_global->oldest_timestamp);
```
Comment (kommiharibabu) [inline src/schema/schema_create.c:85]: It should be `ts`.

File: src/schema/schema_create.c [hunk 3]
```diff
@@ -70,24 +77,25 @@ __check_imported_ts(WT_SESSION_IMPL *session, const char *uri, const char *confi
...
          */
-        if (ckpt->ta.newest_stop_durable_ts > txn_global->oldest_timestamp) {
+        if (ckpt->ta.newest_stop_durable_ts > ts) {
             WT_ASSERT(session, ckpt->ta.newest_stop_durable_ts != WT_TS_MAX);
             WT_ERR_MSG(session, EINVAL,
               "%s: import found aggregated newest stop durable timestamp newer than the current "
-              "oldest timestamp, newest_stop_durable_ts=%" PRIu64 ", oldest_ts=%" PRIu64,
-              uri, ckpt->ta.newest_stop_durable_ts, txn_global->oldest_timestamp);
+              "%s timestamp, newest_stop_durable_ts=%" PRIu64 ", %s_ts=%" PRIu64,
+              uri, ts_name, ckpt->ta.newest_stop_durable_ts, ts_name, txn_global->oldest_timestamp);
```
Comment (kommiharibabu) [inline src/schema/schema_create.c:98]: Here also, it should be `ts`.

File: src/schema/schema_create.c (hunk 2 above)
Comment (quchenhao) [reply src/schema/schema_create.c:85]: oh, my bad. Curious why the complier doesn't catch this.

File: dist/api_data.py (hunk 1 above)
Comment (quchenhao) [reply dist/api_data.py:1394]: I feel `timestamp` is too vague.

**Comment by quchenhao** [comment]
@kommiharibabu @bvpvamsikrishna This is ready now.

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Pull request merged**